src/Microsoft.Health.Dicom.SqlServer/Features/CustomTag/CustomTagDataType.cs

```diff
@@ -0,0 +1,38 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.SqlServer.Features.CustomTag
+{
+    /// <summary>
+    /// Date type of custom tag.
+    /// </summary>
+    public enum CustomTagDataType
```

Comment:
`internal`?

Reply:
And where is this now used?

Reply:
should be internal, suppose only within this project.